rfcs/0013-changes-lifecyle.md

```diff
@@ -0,0 +1,348 @@
+
+When a version is being launched, for example, `1.10` a new section `1.11-nightly` will be created empty. From this time on, all new PRs will create a new item under the "next version", `1.11-nightly` in this example.
+
+Each item will follow the template:
+
+```
+# Version 1.11-nightly
+
+## Forc
+
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+### Security
+
+## Tools
+
+...
+
+## Sway
+
+...
+
+## Standard Libraries
+
+...
+
+# Version 1.10
+
+...
+```
+
+Inside each section, items will follow the template of a user friendly one line description and a link to the PR.
+
+```
+- Index operator using Index trait [#6356](https://github.com/FuelLabs/sway/pull/6356)
+```
+
+Given that all PRs will touch this file, to avoid conflicts and decrease the experience when merging them, we will use "git custom merge driver" ([https://git-scm.com/docs/gitattributes#\_defining_a_custom_merge_driver](https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver)).
+
+This allows a custom merge strategy to a specific file using ".gitattributes".
+
+```
+ReleaseNotes.md merge=union
+```
+
+## Breaking changes
+
+A breaking change is a difference in functionality from the previous version of the compiler that may require an update to Sway code in order for it to compile.
+
+The following changes are defined to be breaking changes and will need to follow the process of being gated by feature flags.
+
+1. Code without deprecated warning to be flagged as an error;
+2. ABI JSON properties to be removed or have their type changed;
+3. Binary encoding that will break how SDK (and others) communicate with the compiler, including:
+   - How scripts/predicates accept arguments on `main`;
+   - How scripts/predicates return data from `main`;
+   - How the `contract method selector` is encoded;
+   - How `contract method` arguments are encoded;
+   - How `log` data is encoded;
+   - How `message` data is encoded;
+4. IR changes;
+5. Receipt parsers to break;
+6. When a compiler feature or a standard library produces different behavior for the same code (semantic changes);
+7. When `storage` is impacted. Particularly addresses.
+
+## Feature flags
+
+Any complex change that needs to be gated will follow these steps.
+
+1. A specific GitHub issue labeled with `feature-*` and `track-feature`. This issue should be the umbrella for everything related to this feature;
+2. A preliminary PR enabling the feature flag should be created and linked to the umbrella issue; This PR will enable the feature flag `--experimental <comma-separated-list>` on all tools.
+3. As many PRs will be created and merged as normal;
+4. A chapter inside `Sway Unstable Book` will be created and updated as needed;
+5. When the feature is ready, a closing PR will be created and wait until the feature flag is enabled by default.
+6. On a later date, the feature flag can be removed making the feature the default behavior of the compiler.
```

Comment:
.. and the docs should be migrated to the stable portion of the book

Reply:
Done.